sys/build.sh

```diff
@@ -95,6 +95,7 @@ unset DEPS
 ./configure ${CFGARG} --prefix=${PREFIX} || exit 1
 ${MAKE} -s -j${MAKE_JOBS} MAKE_JOBS=${MAKE_JOBS} || exit 1
 if [ "`uname`" = Darwin ]; then
+	./sys/macos-cert.sh
```

Comment:
we dont want to generate a new cert on every run

Reply:
we don't, the script terminates at the beginning if the cert is already present
```
security find-certificate -Z -p -c "org.radare.radare2" /Library/Keychains/System.keychain > /dev/null 2>&1
 EXIT_CODE=$?
 if [ $EXIT_CODE -eq 0 ]; then
 	# Certificate has already been generated and installed
 	exit 0
 fi
```